Content.Shared/Tag/TagComponent.cs

```diff
@@ -1,10 +1,12 @@
-﻿using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Set;
+﻿using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Set;
 
 namespace Content.Shared.Tag
 {
-    [RegisterComponent, Access(typeof(TagSystem))]
+    [RegisterComponent, NetworkedComponent, Access(typeof(TagSystem))]
     public sealed partial class TagComponent : Component
     {
+        [AutoNetworkedField]
```

Comment:
This is doing nothing please remove.

Reply:
Gonna remove and test, I'll be back in a bit.

Reply:
Should be resolved now.